### PR TITLE
Updated HTML and CSS files to create "Works" and "About me" sections

### PR DESCRIPTION
**Pull request by uuukiiiyooo**
The next two sections of this project included "Works" and "About me" on the previous HTML file.

- The design was intended to reproduce [this template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=PgJqX5qMbVrq0IQt-0) in a 375px screen width.
- For "Work" section I used a grid so that It'll be easier to rearrange for other screen sizes
- In the "About me" section I used a dropdown structure to display "Languages", "Frameworks" and "Skills" so that it can be interactive in the future.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -132,6 +133,209 @@ body {
...
+        font-weight: 500;
+        color: #396DF2;
+        border: 1px solid #6070FF;
+        border-radius: 8px;
+        background-color: #FFFFFF;
+    }
+
+    .aboutMe {
+        padding: 114px 24px;
+    }
```
Comment (adel-gu) [inline styles.css:238]: - [ ] Please follow the Figma design as a project requirement for making the top-right border-radius as shown in the Figma design for the about section.

### Your Design
image

### Figma Design 
image

- **NOTE ⚠️:** currently the about section background color inherits from the body element background color, so the border-radius for the about section won't show the result until you change the body background color.

File: styles.css [hunk 2]
```diff
@@ -132,6 +133,209 @@ body {
...
+        line-height: 52px;
+        font-weight: 700;
+        margin-top: 0;
+    }
+
+    .aboutPara {
+        margin-top: -7%;
+        font-size: 15px;
+        line-height: 24px;
+    }
```
Comment (adel-gu) [inline styles.css:251]: - [ ] Please follow the same Notion in the comment above about the sizes, colors, and spaces for this section font so it looks the exact same as shown in the Figma design.

### Your Design
image

### Figma Design
image

File: styles.css [hunk 3]
```diff
@@ -132,6 +133,209 @@ body {
...
+        width: 300px;
+        padding: 23px 0px;
+        border-bottom: 1px solid #DFE1E6;
+    }
+
+    /*Adding this class just to visually represent the onclick action*/
+    .dropdownTitleClick {
+        display: block  ;
+        padding: 20px 0 36px 0;
+    }
```
Comment (adel-gu) [inline styles.css:333]: - [ ] For the abilities section please consider adding the icons for the languages, skills, and frameworks so it looks similar to the Figma design.

### Your design:
image
image

### Figma Design
image
image

File: styles.css [hunk 4]
```diff
@@ -132,6 +133,209 @@ body {
...
+        margin: 0;
+        padding: 0;
+        display: flex;
+    }
+
+    .dropdownItem {
+        font-size: 20px;
+        line-height: 24px;
+        font-weight: 500;
+    }
```
Comment (adel-gu) [inline styles.css:296]: - [ ] For the abilities section please consider adding the icons for the languages, skills, and frameworks so it looks similar to the Figma design.

### Your design:
image
image

### Figma Design
image
image

File: styles.css [hunk 5]
```diff
@@ -132,6 +133,209 @@ body {
...
+        justify-content: flex-start;
+        align-items: center;
+        width: 303px;
+        height: 67px;
+        background-color: #f7f7f9;
+        border-radius: 8px;
+        font-size: 15px;
+        line-height: 20px;
+        font-weight: 500;
+    }
```
Comment (adel-gu) [inline styles.css:315]: - [ ] Please ensure that the skill item should fill the full width of the screen as shown in the Figma design

### Your Design 
image

### Figma Design
image

**Review by adel-gu — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES ♻️ 

**Hi @uuukiiiyooo 👋🏼** 

Such an effort and Good job so far!  👏👏💪 You really implemented this project pretty well, However, am afraid that there are some issues that you still need to work on to go to the next project but you are almost there!  👌🚀🚀🚀

## Highlights
- All linter check are passing ✔️ 
- Good commit history ✔️ 
- HTML/CSS best practices are well used ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @uuukiiiyooo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

200w


## Highlights
Great effort in making the requested changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**